### PR TITLE
15 change array to brackets

### PR DESCRIPTION
**Pull request by digitaldiina**
Turned out to be way more than a few occurrences

### CODE REVIEW

File: docs/api-reference/classes/NSActivityFilter/GetFutureDateList.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+title: MDOListItem[] GetFutureDateList()
 intellisense: NSActivityFilter.GetFutureDateList
 keywords: NSActivityFilter, GetFutureDateList
 so.topic: reference
 ---
 
 The list of future date selections. This is a static MDO list with standard values.
 
-**Returns:** MDOListItemArray
+**Returns:** MDOListItem[]
```
Comment (kronicz) [inline docs/api-reference/classes/NSActivityFilter/GetFutureDateList.md:11]:
```suggestion
**Returns:** NSMDOListItem[]
```

File: docs/api-reference/classes/NSActivityFilter/GetFutureDateList.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
 
 ```crmscript
 NSActivityFilter thing;
-MDOListItemArray futureDateList  = thing.GetFutureDateList();
+MDOListItem[] futureDateList  = thing.GetFutureDateList();
```
Comment (kronicz) [inline docs/api-reference/classes/NSActivityFilter/GetFutureDateList.md:16]:
```suggestion
NSMDOListItem[] futureDateList  = thing.GetFutureDateList();
```

File: docs/api-reference/classes/NSActivityFilter/GetGroupList.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
 ﻿---
 uid: crmscript_ref_NSActivityFilter_GetGroupList
-title: SelectableMDOListItemArray GetGroupList()
+title: SelectableMDOListItem[] GetGroupList()
```
Comment (kronicz) [inline docs/api-reference/classes/NSActivityFilter/GetGroupList.md:3]:
```suggestion
title: NSSelectableMDOListItem[] GetGroupList()
```

File: docs/api-reference/classes/NSActivityFilter/GetGroupList.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
 intellisense: NSActivityFilter.GetGroupList
 keywords: NSActivityFilter, GetGroupList
 so.topic: reference
 ---
 
 The list of groups to filter on
 
-**Returns:** SelectableMDOListItemArray
+**Returns:** SelectableMDOListItem[]
```
Comment (kronicz) [inline docs/api-reference/classes/NSActivityFilter/GetGroupList.md:11]:
```suggestion
**Returns:** NSSelectableMDOListItem[]
```

File: docs/api-reference/classes/NSActivityFilter/GetGroupList.md [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
 
 
 ```crmscript
 NSActivityFilter thing;
-SelectableMDOListItemArray groupList  = thing.GetGroupList();
+SelectableMDOListItem[] groupList  = thing.GetGroupList();
```
Comment (kronicz) [inline docs/api-reference/classes/NSActivityFilter/GetGroupList.md:16]:
```suggestion
NSSelectableMDOListItem[] groupList  = thing.GetGroupList();
```

**Review by kronicz — CHANGES_REQUESTED** [review]

**Review by kronicz — COMMENTED** [review]
Looks correct with changes added.

**Review by kronicz — APPROVED** [review]

**Review by SuperOfficeDevNet — APPROVED** [review]

**Pull request merged**